### ISSUE TITLE
集成之后导致cellForRow方法不调用

### ISSUE THREAD

**Issue by SunQiDreamer** [tigerAndBull/TABAnimated#134]
### New Issue Checklist

* [ ] I have read the [Documentation](https://github.com/tigerAndBull/TABAnimated/tree/master/Documents)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/TABAnimated/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. iOS / macOS
 Platform Version        | e.g. 12.0 / 10.14.0
 TABAnimated Version      | e.g. 5.0.0 / 4.4.0
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by tigerAndBull** [comment]
如果其他第三方库，诸如RxSwift，QMUI等，这些库hook了tableView的数据源和数据代理，会出现这样的问题。

**Comment by tigerAndBull** [comment]
已关闭此issue，如对回复存在疑惑，可以重新开启

**Comment by tigerAndBull** [comment]
2.5.2已适配第三方库，如QMUI，RxSwift.
